httpstan/views.py

```diff
@@ -26,6 +26,10 @@
 logger = logging.getLogger("httpstan")
 
 
+# match a string such as `Iteration: 2000 / 2000 [100%]  (Sampling)`
+iteration_info_re = re.compile(rb"Iteration:\s+\d+ / \d+ \[\s*\d+%\]\s+\(\w+\)")
```

Comment:
Should there be `\s*` instead of real spaces in `\d+ / \d+ \[`?

Also `\w` or `\S`

Reply:
I think this is correct. You can see how the messages are created in `src/stan/services/util/generate_transitions.hpp`:

```
      message << "Iteration: ";
      message << std::setw(it_print_width) << m + 1 + start << " / " << finish;
      message << " [" << std::setw(3)
              << static_cast<int>((100.0 * (start + m + 1)) / finish) << "%] ";
      message << (warmup ? " (Warmup)" : " (Sampling)");
```